ubpf/assembler.py

```diff
@@ -1,6 +1,10 @@
-from asm_parser import parse, Reg, Imm, MemRef
+from .asm_parser import parse, Reg, Imm, MemRef
 import struct
-import StringIO
+import six
+if six.PY2:
+    from StringIO import StringIO as io
+else:
+    from io import BytesIO as io
```

Comment:
What about the following instead of relying on `six`?
```python
try:
    from StringIO import StringIO as io
except ModuleNotFoundError:
    from io import BytesIO as io
```